cmd/cdi-importer/importer.go

```diff
@@ -117,6 +118,16 @@ func main() {
 				}
 				os.Exit(1)
 			}
+		case controller.SourceImageio:
```

Comment:
As for registry imports, let's make sure this only is allowed with the kubevirt content type.  Please remember to update the support matrix in https://github.com/kubevirt/containerized-data-importer/blob/master/doc/supported_operations.md

Reply:
Fixed in #1151